dashboard/urls.py

```diff
@@ -16,11 +16,13 @@
 from django.conf.urls import url, include
 from django.contrib import admin
 
-from .views import Home
+from .views import Home, VapidValidation
 
 urlpatterns = [
     # our app urls
     url(r'^$', Home.as_view(), name='home'),
+    url(r'^vapid/validate/', VapidValidation.as_view(),
+        name='vapid_validation'),
```

Comment:
As mentioned in [another comment](https://github.com/mozilla-services/push-dev-dashboard/pull/62/files#r54302784), we should move this view to the `push/views.py` module, so we would change this to match ... 
```
from push.views import PushApplication
...
url (r'^/push/(?P<pk>[^/.]+)', PushApplication.as_view(), name='push-app-details')
```